src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -140,22 +149,39 @@ public void AssignRowsAndInitializeDisplay()
             foreach (ObservedProvider provider in _providers.Values.OrderBy(p => p.KnownProvider == null).ThenBy(p => p.Name)) // Known providers first.
             {
                 Console.WriteLine($"[{provider.Name}]"); row++;
+                int rowCount = 0;
+
                 foreach (ObservedCounter counter in provider.Counters.Values.OrderBy(c => c.DisplayName))
                 {
+                    if(rowCount >= _consoleHeight - 5)
```

Comment:
Maybe choosing a value that isn't hard coded would be a good idea, in case the header changes at some point?

Reply:
This value is no longer needed with changes made earlier